anime_js/src/lib.rs

```diff
@@ -0,0 +1,99 @@
+use std::collections::BTreeMap;
+use std::sync::Once;
+
+use anime::Anime;
+use anyhow::Result;
+use geo::{GeometryCollection, LineString};
+use geojson::de::deserialize_geometry;
+use serde::{Deserialize, Serialize};
+use utils::Mercator;
+use wasm_bindgen::prelude::*;
+
+static START: Once = Once::new();
+
+/// Takes two GeoJSONs, matches LineStrings, and returns the TODO   optional index of the best matching
```

Comment:
Actually, please hold off review -- I was about fix these docs, but then I've gotten quite confused about the returned value. I've been expecting a mapping from target to matching source candidates, but it appears the other way.